### PR TITLE
Homework

### PR DESCRIPTION
**Pull request by Sh1co**

### CODE REVIEW

File: Assets/Scripts/Movement.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{
+
+    [SerializeField] private float _movementSpeed = 5.0f;
+    [SerializeField] private float _acceleration = 7.0f;
+    [SerializeField] private float _breaking = 15.0f;
+    [SerializeField] [Range(0.01f, 1.0f)] private float _rotationSpeed = 0.15f;
+    
+    void Update()
+    {
+        _moveHorizontal = Input.GetAxisRaw ("Horizontal");
```
Comment (Delt06) [inline Assets/Scripts/Movement.cs]: why make them fields if they are overwritten each frame?

File: Assets/Scripts/Movement.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        _moveVertical = Input.GetAxisRaw ("Vertical");
+
+        if (_moveHorizontal != 0 || _moveVertical != 0)
```
Comment (Delt06) [inline Assets/Scripts/Movement.cs]: float equality checks are not reliable, you should use Mathf.Approximately

File: Assets/Scripts/Movement.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        {
+            _movementDir = new Vector3(_moveHorizontal, 0.0f, _moveVertical);
+            _movementDir = _movementDir.normalized;
+            transform.rotation = Quaternion.Slerp(transform.rotation, Quaternion.LookRotation(_movementDir), _rotationSpeed);
```
Comment (Delt06) [inline Assets/Scripts/Movement.cs:22]: no deltaTime

Comment (Delt06) [reply Assets/Scripts/Movement.cs:22]: -0.5 points

**Review by Delt06 — APPROVED** [review]
## General Feedback
Very nice and responsive controls

## Grade
4.5 + 1 (extra) + 0.5 (code quality) = 6 points

File: Assets/Scripts/Movement.cs (hunk 3 above)
Comment (Sh1co) [reply Assets/Scripts/Movement.cs:22]: How do I use deltaTime with Quaternion.Slerp

**Pull request merged**